contract-examples/contracts/IFeeManager.sol

```diff
@@ -0,0 +1,32 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2;
```

Comment:
Can we update this and everything else to min version of v0.8.15?

Reply:
Alternatively, let's move this to at least v0.8.0 and have a separate PR to bump everything to v0.8.15.